src/mixins/window-width-mixin.js

```diff
@@ -0,0 +1,34 @@
+export default {
+  data: () => {
+    return {
+      windowWidth: 0,
+      respondToResize: null,
+    }
+  },
+  mounted() {
+    this.windowWidth = window.innerWidth;
+    
+    window.addEventListener('resize', getResizeFunction());
+  },
+  destroyed() {
+    window.removeEventListener('resize', getResizeFunction());
+  },
+  methods: {
+    getResizeFunction() {
+      let resizeFunction = () => {
+        // timeout reduces the frequency of resize events
+        clearTimeout(respondToResize);
```

Comment:
You'll have to declare this variable somewhere. I recommend it as a `data` property. And can you name it `resizeTimeout`?

Reply:
The timeout is called respondToResize and it is in the data property.  Not sure what your getting at.